web/sites/default/config/default/block.block.champions_accessorganizationmembers.yml

```diff
@@ -0,0 +1,29 @@
+uuid: db70f392-5a4c-4537-9da3-77adbe46c7db
+langcode: en
+status: true
+dependencies:
+  config:
+    - facets.facet.access_organization_members
+  module:
+    - facets
+    - system
+  theme:
+    - champions
+id: champions_accessorganizationmembers
+theme: champions
+region: primary_sidebar
+weight: 5
+provider: null
+plugin: 'facet_block:access_organization_members'
+settings:
+  id: 'facet_block:access_organization_members'
+  label: Organization
+  label_display: visible
+  provider: facets
+  context_mapping: {  }
+  block_id: champions_accessorganizationmembers
+visibility:
+  request_path:
+    id: request_path
+    negate: false
+    pages: /members
```

Comment:
@protitude eventually there will be a faceted view on Campus Champions at /champions/current-campus-champions but not a /members page there.